src/renderer/components/top-nav/top-nav.js

```diff
@@ -69,7 +69,7 @@ export default Vue.extend({
     },
 
     backwardText: function () {
-      return this.$t('Backward')
```

Comment:
would like to see this reverted to Backward

Reply:
There's currently no translation for "Backward" in the locale file but there is one for "Back" and it's in relation to the nav-bar. https://github.com/FreeTubeApp/FreeTube/blob/1e1a04c81f256384071a520fd78f5146711810db/static/locales/en-US.yaml#L29